tests/run_tests_in_background.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+
+"$@"
+echo $? > /tmp/jupyter/tests/rc
```

Comment:
I don't think this is possible to implement inline, so I had to create this simple bash script. If someone knows how to do it better, please, tell me.